examples/source/0.introduction/AMPHTML_Email_Hello_World.html

```diff
@@ -54,66 +47,33 @@
-    <div class="images">
-      <amp-img src="https://ampbyexample.com/img/amp.jpg" width="800" height="600" layout="responsive"></amp-img>
-      <!--
-        ### AMP Components
-
-        The following AMP components are supported in AMPHTML email messages.
-        **Dynamic Content </h4>
-        <ul>
-          <li>[`<amp-form>`](https://ampbyexample.com/components/amp-form/)</li>
-          <li>[`<amp-selector>`](https://ampbyexample.com/components/amp-selector/)</li>
-          <li>[`<amp-bind>` and `<amp-state>`](https://ampbyexample.com/components/amp-bind/)</li>
-          <li>[`<amp-list>`](https://ampbyexample.com/components/amp-list/)</li>
-          <li>[`<template type="amp-mustache">`](https://ampbyexample.com/components/amp-mustache/)</li>
-        </ul>
-        <h4>Layout</h4>
-        <ul>
-          <li>[`<amp-accordion>`](https://ampbyexample.com/components/amp-accordion/)</li>
-          <li>[`<amp-carousel>`](https://ampbyexample.com/components/amp-carousel/)</li>
-          <li>[`<amp-sidebar>`](https://ampbyexample.com/components/amp-sidebar/)</li>
-          <li>[`<amp-image-lightbox>`](https://ampbyexample.com/components/amp-image-lightbox/)</li>
-          <li>[`<amp-lightbox>`](https://ampbyexample.com/components/amp-lightbox/)</li>
-          <li>[`<amp-fit-text>`](https://ampbyexample.com/components/amp-fit-text/)</li>
-          <li>[`<amp-timeago>`](https://ampbyexample.com/components/amp-timeago/)</li>
-        </ul>
-        <h4>Media</h4>
-        <ul>
-          <li>[`<amp-img>`](https://ampbyexample.com/components/amp-img/)</li>
-          <li>[`<amp-anim>`](https://ampbyexample.com/components/amp-anim/)</li>
-        </ul>
+    <h1 class="helloworld">Hello!</h1>
+    <!--
+      Certain tags, such as the `<img>` tag, are replaced with equivalent or slightly enhanced custom AMP HTML tags (see [HTML Tags in the specification](https://github.com/ampproject/amphtml/blob/master/spec/amp-html-format.md)).
+    -->
+    <amp-img src="https://ampbyexample.com/img/amp.jpg" width="800" height="600" layout="responsive"></amp-img>
+    <!--
+    Important: URLs most use absolut paths in AMPHTML email.
 
-        Since we imported the `amp-carousel` component in the header as example, let's try it here.
-      -->
+    Since we imported the `amp-carousel` component in the header as example, let's try it here.
+    -->
     <amp-carousel width="800" height="600" layout="responsive" type="slides">
-        <amp-img src="https://ampbyexample.com/img/image1.jpg" width="800" height="600" alt="a sample image"></amp-img>
-        <amp-img src="https://ampbyexample.com/img/image2.jpg" width="800" height="600" alt="another sample image"></amp-img>
-        <amp-img src="https://ampbyexample.com/img/image3.jpg" width="800" height="600" alt="and another sample image"></amp-img>
-      </amp-carousel>
-    </div>
+      <amp-img src="https://ampbyexample.com/img/image1.jpg" width="800" height="600" alt="a sample image"></amp-img>
```

Comment:
https://ampbyexample.com/img/image1.jpg
```suggestion
      <amp-img src="<% hosts.platform %>/static/samples/img/image1.jpg" width="800" height="600" alt="a sample image"></amp-img>
```

Not crucial, since I need someone to go over all URLs again as soon as we have unprefixed statics.